### PR TITLE
Namingsystem null

### PR DESCRIPTION
**Pull request by daco101**
- [x] update package.json with cqm-models version once gems are published

This PR updates the engine to test the naming system null difference, and includes changes to support the new converter and models branch (so the pretty print results stay consistent).

Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1668
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @mayerm94 @losborne @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: spec/models/calculator_spec.js [hunk 1]
```diff
@@ -600,4 +601,19 @@ describe('Calculator', () => {
...
+      const valueSetsByOid = getJSONFixture('measures/CMS108v7/value_sets.json');
+      const measure = getJSONFixture('measures/CMS108v7/CMS108v7.json');
+      const patients = [getJSONFixture('patients/CMS108v7/INR4no_decimal_DayOfAnes_NUMERPass.json')];
+      const options = { doPretty: true };
+      const calculationResults = Calculator.calculate(measure, patients, valueSetsByOid, options);
+      const result = Object.values(calculationResults[Object.keys(calculationResults)[0]])[0];
+
+      const sampleResult = result['statement_results']['VenousThromboembolismProphylaxis']['Is In Low Risk for VTE or On Anticoagulant'];
+      const sampleResultId = sampleResult.raw[0].id.value;
+      expect(sampleResult['pretty']).toEqual(`[{\n  id: {\n    value: "${sampleResultId}",\n    namingSystem: null\n  },\n  authorDatetime: 11/01/2012 10:00 AM\n}]`);
```
Comment (mayerm94) [inline spec/models/calculator_spec.js]: Test description is namingsystem is not null, yet the test expects the namingSystem to be null

File: spec/fixtures/json/patients/CMS108v7/INR4no_decimal_DayOfAnes_NUMERPass.json [hunk 2]
```diff
@@ -0,0 +1,404 @@
+{
```
Comment (mayerm94) [inline spec/fixtures/json/patients/CMS108v7/INR4no_decimal_DayOfAnes_NUMERPass.json]: This patient should be added to bonnie-fixtures to make the re-generation of this fixture easier
